docs/configuration.md

```diff
@@ -97,6 +97,18 @@ storage:
 		poolname: radospool
 		username: radosuser
 		chunksize: 4194304
+	swift:
+		username: username
+		password: password
+		authurl: https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth
```

Comment:
Hi @nevermosby : we tested against SoftLayer Object Storage endpoints also as shown at http://sldn.softlayer.com/article/Introduction-Object-Storage, for example, https://lon02.objectstorage.softlayer.net/auth/v1.0/ . Hence I put in that comment, but not sure whether or not that is keystone v1/v2/v3, it might be just proprietary. But we tested the swift driver against that and it works. Hence I put in that comment.